zipline/utils/run_algo.py

```diff
@@ -171,13 +167,12 @@ def choose_loader(column):
         env = None
         choose_loader = None
 
-    if live_trading:
+    if broker:
         start = pd.Timestamp.utcnow()
         end = start + pd.Timedelta('1', 'D')
-    TradingAlgorithmClass = (partial(LiveTradingAlgorithm,
-                                     live_trading=live_trading,
-                                     broker=broker)
-                             if live_trading else TradingAlgorithm)
+
+    TradingAlgorithmClass = (partial(LiveTradingAlgorithm, broker=broker)
```

Comment:
The LiveTradingAlgorithm class needs to be adopted too as it is still trying to store live_trading parameter.